about/acknowledgements.md

```diff
@@ -3,6 +3,15 @@ layout: page
 title: Acknowledgments
 ---
 
+#### Primary Contributors
+
+This material, including lectures, website design and implementation, and
```

Comment:
How about just: "Course materials and website design have been primarily developed and implemented by"?